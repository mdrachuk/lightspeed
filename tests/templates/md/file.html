<html lang="en">
<head><title>{{ source.file }}</title></head>
<body>
file: {{ source.file }}
file.name: {{ source.file.name }}
file.extension: {{ source.file.extension }}

</body>
</html>