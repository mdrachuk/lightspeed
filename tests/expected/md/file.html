<html lang="en">
<head><title>plain.md</title></head>
<body>
filename: plain.md
filename.name: plain
filename.extension: .md

</body>
</html>